docs/cli.md

```diff
@@ -0,0 +1,384 @@
+
+GUI Equivalent:
+
+<img src="images/cli013.png" width="768px" alt="rm gui example"/>
+
+
+## cd
+`cd` stands for "<b>c</b>hange <b>d</b>irectory". It switches you to the specified directory. [More info](https://man7.org/linux/man-pages/man1/cd.1p.html).
+
+CLI Example:
+
+<img src="images/cli008.png" width="768px" alt="cd cli example"/>
+
+GUI Equivalent:
+
+<img src="images/cli014.png" width="768px" alt="cd gui example"/>
+
+
+## mv 
+`mv` is short for "<b>m</b>o<b>v</b>e". It moves the specified file or directory to the specified path.
+
+`mv` is also commonly used to rename a file or directory. [More info](https://man7.org/linux/man-pages/man1/mv.1.html).
+
+CLI Example (moving):
+
+<img src="images/cli009.png" width="768px" alt="mv cli example"/>
+
+Note the `/` at the end of `stuff/`. While not necessary, it is convention to end directory paths with `/`.
+
+GUI Equivalent (moving):
+
+<video controls="controls" width="768px" name="mv gui example">
+  <source src="images/cli_vid001.mov">
+</video>
+
+CLI Example (renaming):
+
+<img src="images/cli010.png" width="768px" alt="mv cli example"/>
+
+GUI Equivalent (renaming):
+
+<img src="images/cli015.png" width="768px" alt="mv gui example"/>
+
+## cp 
+`cp` is short for "<b>c</b>o<b>p</b>y". It copies the specified file or directory to the specified path.
+[More info](https://man7.org/linux/man-pages/man1/cp.1.html).
+
+CLI Example:
+
+<img src="images/cli011.png" width="768px" alt="cp cli example"/>
+
+GUI Equivalent:
+
+<img src="images/cli016.png" width="768px" alt="cp gui example"/>
+
+
+## open / wslview
+On MacOS, `open` opens a file or directory with the default application.
+[More Info](https://ss64.com/osx/open.html).
+
+On WSL (Windows), `wslview` opens a file or directory with the default application.
+[More Info](https://wslutiliti.es/wslu/man/wslview.html).
+
+CLI Example:
+
+<video controls="controls" width="768px" name="mv gui example">
+  <source src="images/cli_vid002.mov">
+</video>
+Note the `.` path used. This is a [special path](#special-paths) that refers to the current directory.
+
+GUI Equivalent:
+
+<img src="images/cli017.png" width="768px" alt="open gui example"/>
+
+
+## Special Paths
+A path is the location of a given file or directory in your computer. There are two types of paths: absolute and relative.
+
+<b>Absolute path</b>: a path that starts from the [root directory](#-3). eg. `/Users/ohjun/Desktop/project/`
+
+<b>Relative path</b>: a path that starts from the current directory.
+
+You can choose the type of path depending on the situation, but relative pathing work in most cases and requires less typing.
+
+Below are some common, useful, special paths:
+
+### .
+`.` refers to the current directory. Check the [open/wslview](#open--wslview) section for an example.
+
+### ..
+`..` refers to the parent directory (of the current directory).
+
+<img src="images/cli018.png" width="768px" alt=".. example"/>
```

Comment:
I'd use a `console` block here instead of a screenshot.  Same for the other "terminal only" screenshots.